### DISCUSSION TITLE
Which blockchain would you like to have in UW?

### DISCUSSION THREAD

**Discussion by serrq** [horizontalsystems/unstoppable-wallet-android#6201] [Polls]
Basically what the title asks.

**Comment by serrq** [comment]
If your choise doesn't fit no result in list please write down here your suggestion and I will add it/them in list.

**Reply by NatureHog** [reply]
I already voted, but I think Dogecoin would be interesting to see...

**Reply by serrq** [reply]
It seems impossible adding a new one. Attempts to add it all failed.

**Reply by NatureHog** [reply]
No problem, as long as it's at least considered... :-)

**Reply by serrq** [reply]
Ok. Dogecoin added. Please vote again because adding a new one cause reset.

**Reply by 3point14159** [reply]
dogecoin: purely fueled by memeology, not innovation, and just look at that market cap compared to other coins actually moving products in realspace!

Proof you don't actually have to provide any real value to find suckers to buy your dogecrap.

Wouldn't be surprised if it was a just another play from Wall Street to siphon even more money from people.

**Reply by serrq** [reply]
@NatureHog Please vote again. An other token has been added (max 8 allowed). Sorry for the inconvenience.

**Comment by 3point14159** [comment]
I've been really digging OXEN as of late. They are behind some great freedom tech that's immersed in blockchain (Monero specifically) and making privacy friendly tools that are (and here's the kicker) easy for anyone to use. This is what's been lacking with the tech; achieving the "network effect". If you make the tech easy enough to pick-up and use, with privacy optimized as default (look at the better days of Signal), the users will come. I believe their team are dedicated to seeing their vision and mission though to completion and love using their wares.

https://oxen.io/
https://getsession.org/
https://lokinet.org/
https://github.com/oxen-io

I am refocusing my assets towards projects that are actually making things happen. Too much money thrown at who can make the highest tx throughputs and not enough on the ones that actually add human value.

**Reply by serrq** [reply]
I find Tezos exactly what you mean.

They go on to create value, increase network speed, make operations easy on DEX, and deliver services based on WEB3.

**Reply by 3point14159** [reply]
That's exactly what I don't mean.

"create value, increase network speed, make operations easy on DEX, and deliver services based on WEB3."

Can't you say the same thing about any other crapcoin?

What can I buy with Tezos? Are they present on any marketplace? Are they developing any other technology that extends from their blockchain? Other than just claiming specks, what are they actually doing?

**Reply by serrq** [reply]
This section is called "Discussions", not "Aggressions". Please make your choice serenely in the respect the ideas of others.

**Reply by 3point14159** [reply]
@serrq 

Aggression? I believe you were the one misunderstanding what I was saying, I was merely clarifying what you _thought_ I meant.

Some consider publicly misrepresenting other people's statements as "_aggressions_", did you think about that?

I think I made my choice peacefully and constructively, what are you even referring to?

Don't be so easily triggered; it's distasteful.

**Reply by 3point14159** [reply]
Oh, and there's this:

A Tale of Two ICOs: How Maidsafe, Tezos Endangered Own Projects With Critical Mistakes

https://cointelegraph.com/news/a-tale-of-two-icos-how-maidsafe-tezos-endangered-own-projects-with-critical-mistakes

Uhhhmmmm, yeeeeeah; I think I am going to stay away from Tezos.

Seems like there are so many crapcoins that simply exist to siphon legit assets away from those who don't know any better.

Don't be fooled out there people and don't let other's do your thinking/due diligence for you.

**Reply by serrq** [reply]
@3point14159 Oxen added. Enjoy your vote. Peace.

**Reply by 3point14159** [reply]
@serrq Thanks for the love. I done voted long ago; chose XMR out of the options that were there. Really it's the same vote either way seeing that OXEN is a fork of Monero.

Sorry if I was a bit snipey, been in a sad/frustrated/sick space the past few days, and things have been coming out of me that shouldn't.

**Reply by serrq** [reply]
Don’t worry. I get it.

Unfortunately, every time a new currency is added, the previous votes reset.

Therefore, as a community, we need your choice. We are in full democracy.

**Reply by 3point14159** [reply]
Vote cast. Thanks for the consideration.
